spec/dbModels.ts

```diff
@@ -163,9 +163,7 @@ describe('Database Models', () => {
         password: 'password'
       });
 
-      await user.save(err => {
-        return err;
-      });
+      await user.save();
 
       await user.comparePassword('password', (err: any, isMatching: boolean) => {
         chai.assert.isTrue(isMatching);
```

Comment:
I'd probably import assert from chai and then just use assert.isTrue not to type chai everywhere.
The current usage is correct, just bringing up an idea.